components/prism-css-extras.js

```diff
@@ -10,7 +10,19 @@ Prism.languages.css.selector = {
 };
 
 Prism.languages.insertBefore('css', 'function', {
+	'variable': {
+		pattern: /(var\()[^)]+(?=\))/,
```

Comment:
This pattern will also include [possible default values](https://www.w3.org/TR/css-variables/#using-variables).

But we can use the naming convention of [custom properties](https://www.w3.org/TR/css-variables/#defining-variables) to make a simple pattern which matches all variables accessed via `var` or not.

> A custom property [= variable] is any property whose name starts with two dashes [...].

Using that, `/(^|[^-\w\xA0-\uFFFF])--[-_a-z\xA0-\uFFFF][-\w\xA0-\uFFFF]*/i` should do the job.

Depending on whether we insert it before `property` we can also highlight the custom property declarations.

Reply:
Gah, hit merge too quickly!

Reply:
It's alright, I review too late.